Makefile

```diff
@@ -15,13 +15,18 @@ format:	flake black isort mypy # run all formatters at once
 generate-req:
 	poetry export --without-hashes --format=requirements.txt > requirements.txt
 
-build-dockers:
-	docker-compose build
-
-run-docker:
-	docker-compose -f docker-compose.yaml up
-
-run-local:
-	poetry run uvicorn app.main:app --proxy-headers --host 0.0.0.0 --port 8000
-
-run-clean: generate-req run-docker
\ No newline at end of file
+run-clean: generate-req run-docker
+
+run-cloud-build:
+	docker-compose --file docker-compose-cloud.yaml --project-name "visual-similarity-search-cloud" \
+	up -d --no-deps --build ;\
+	sleep 30 ;\
```

Comment:
They allow for data to finish being loaded from the cloud to local volumes before application states are being run for the first time. 

The general issue is that (at this point) Qdrant is not a service that is managed in real-time by any service manager, e.g. Kubernetes. Therefore, it has to be created and fed with data every time a docker-compose process is run. Because of that, volumes for data may not exist as well and their rebuild takes time. Another issue is that it is undefinable from the level of `docker-compose.yaml` for `interactive` container to wait for Qdrant to finish feeding data and restarting.